pkg/kubernetes/components.go

```diff
@@ -35,7 +36,21 @@ func PrintComponents(name, outputFormat string) error {
 			return nil, err
 		}
 
-		return client.ComponentsV1alpha1().Components(meta_v1.NamespaceAll).List(meta_v1.ListOptions{})
+		list, err := client.ComponentsV1alpha1().Components(meta_v1.NamespaceAll).List(meta_v1.ListOptions{})
+		if err != nil {
```

Comment:
Better to use `"k8s.io/apimachinery/pkg/api/errors"` and `errors.IsNotFound(err)`